doc/release/1.15.0-notes.rst

```diff
@@ -261,41 +292,41 @@ Dates, times, and timedeltas can now be histogrammed. The bin edges must be
 passed explicitly, and are not yet computed automatically.
 
 ``histogram`` "auto" estimator handles limited variance better
-------------------------------------------------------------------------
-No longer does an IQR of 0 result in `n_bins=1`, rather the number of bins
+--------------------------------------------------------------
+No longer does an IQR of 0 result in ``n_bins=1``, rather the number of bins
 chosen is related to the data size in this situation.
 
-``histogram`` and ``histogramdd`` return edges matching the float type of the data
-----------------------------------------------------------------------------------
+``histogram`` and ``histogramdd`` edges match the float type of the data
```

Comment:
`return` was a useful disambiguator from the input edges argument. I think this was more correct before, even if the line was long.